src/go/k8s/apis/redpanda/v1alpha1/redpanda_clusterspec_types.go

```diff
@@ -182,11 +182,11 @@ type TieredStoragePersistentVolume struct {
 
 // PersistentVolume
 type PersistentVolume struct {
-	Annotations  map[string]string `json:"annotations"`
+	Annotations  map[string]string `json:"annotations,omitempty"`
 	Enabled      bool              `json:"enabled"`
-	Labels       map[string]string `json:"labels"`
-	Size         string            `json:"size"`
-	StorageClass string            `json:"storageClass"`
+	Labels       map[string]string `json:"labels,omitempty"`
+	Size         string            `json:"size,omitempty"`
```

Comment:
Size isn't required?

Reply:
Essentially this is a values file, so we capture whatever changes we want, otherwise filling this out will become lengthy. Now, maybe we want this?